frontend/src/components/createdProduct/CreatedProduct.tsx

```diff
@@ -19,32 +24,52 @@ export const CreatedProduct = ({ item, profileID }: CreatedProductProps) => {
             await dispatch(fetchUserProducts(profileID));
         }
     };
-
     return (
         <div className={styles.container}>
             <div className={styles.imageWrapper}>
                 <img className={styles.image} src={item.productImageUrl} alt="product" />
-                {activeProfile === item.ownerID && (
-                    <div className={styles.imageHoverSection}>
-                        <div className={styles.imageHoverContainer}>
-                            <button
-                                className={styles.deleteButton}
-                                type="button"
-                                onClick={handleDelete}
-                            >
-                                Delete
-                            </button>
-                            <div className={styles.startAuction}>
-                                <button type="button" className={styles.startAuctionButton}>
-                                    <span>Start auction!</span>
+                {activeProfile === item.ownerID &&
```

Comment:
suggestion: move this expression into a variable for readability, could be called something like `isOwner`